addons/spectator/CfgFunctions.hpp

```diff
@@ -0,0 +1,21 @@
+class CfgFunctions
+{
+    class A3_Expansion_A
+    {
+        class Spectator
+        {
+            class EGSpectator
+            {
+                file = QPATHTOF(functions\DOUBLES(fnc,EGSpectator).sqf);
+            };
+            class EGSpectatorCore
+            {
+                file = QUOTE(A3\Functions_F_Exp_A\EGSpectator\fn_EGSpectator.sqf);
+            };
+            class RscDisplayEGSpectatorCore
+            {
+                file = QUOTE(A3\Ui_F\Scripts\Gui\rscdisplayegspectator.sqfc);
```

Comment:
@veteran29 alternative option to this is to use:
```SQF
call { #include "A3\Ui_F\Scripts\Gui\rscdisplayegspectator.sqf" };
```
instead of
```SQF
call BIS_fnc_RscDisplayEGSpectatorCore;
```
But we would need to have whole file in include section, so I think this is better option (and surprisingly, `sqfc` works, while would not work in include).

Reply:
Function is preferable due to SQFC.